doc/index.html

```diff
@@ -1,34 +1,35 @@
 <section class="landing">
     <div class="hero is-fullheight-with-navbar"> <!-- has-background-primary is-light"> -->
     <div class="hero-body">
-      
+
       <div class="columns is-touch">
         <div class="column is-3 is-3-widescreen is-offset-1-widescreen landing-col-1">
-  
+
           <h1 class="hero-title title">
             <span class="hero-title-1">
-              Fast & Simple 
+              Fast & Simple
             </span>
             <br/>
             <span class="hero-title-2">
-                Tensor Learning 
+                Tensor Learning
             </span>
             <br>
             <span class="hero-title-1">
-              In Python 
+              In Python
             </span>
           </h1>
-          
+
           <div class="content tensorly-functionalities">
             <ul>
-                <li> Open Source, BSD Licensed</li> 
+                <li> Open Source, BSD Licensed</li>
                 <li> Pure Python, Tested & Optimized </li>
                 <li> Flexible Backends for:
                         <a target="_blank" href="https://numpy.org/"> NumPy</a>,
                         <a target="_blank" href="https://pytorch.org/"> PyTorch</a>,
                         <a target="_blank" href="https://www.tensorflow.org/"> TensorFlow</a>,
                         <a target="_blank" href="https://jax.readthedocs.io/en/latest/"> JAX</a>,
-                        and <a target="_blank" href="https://cupy.dev/"> CuPy </a> </li>
+                        <a target="_blank" href="https://cupy.dev/"> CuPy </a>,
+                        and <a target="_blank" href="https://www.paddlepaddle.org.cn/en/install/quick?docurl=/documentation/docs/en/develop/install/pip/linux-pip_en.html"> PaddlePaddle</a> </li>
```

Comment:
Can we link to `https://www.paddlepaddle.org.cn` to be consistent with the other tools?

Reply:
Done